static/pluto

```diff
@@ -0,0 +1,3 @@
+auth		required	pam_radius_auth.so
+account		required	pam_radius_auth.so
```

Comment:
whitespace funkyness

Reply:
weird, since vim has 8 spaces per tab, and that's also the github viewer default. oh well, i converted the file to spaces.